private-templates-service/client/src/utils/versionUtils.tsx

```diff
@@ -0,0 +1,31 @@
+// Time constants
+const month = 30 * 24 *  60 * 60 * 1000;
+const day = 24 * 60 * 60 * 1000;
+const hour = 60 * 60 * 1000;
+const minute = 60 * 1000;
+
+/**
+ * @function
+ * Creates readable amount of time ago string given backend provided Date string 
+ * eg. 2 months ago, 2 days ago, 2 hours ago, 2 minutes ago
+ * @param date 
+ */
+export function getDateString(date: string): string {
+    let currentDate = new Date();
+    let newDate = new Date(date);
+    let difference = Math.abs(currentDate.getTime() - newDate.getTime());
```

Comment:
The result here is in milliseconds, which means this will only work for ~24 days. 2147483647/86400000